providers/dns/liquidweb/liquidweb.go

```diff
@@ -0,0 +1,139 @@
+package liquidweb
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/go-acme/lego/v3/challenge/dns01"
+	"github.com/go-acme/lego/v3/platform/config/env"
+	lwclient "github.com/liquidweb/liquidweb-go/client"
+	"github.com/liquidweb/liquidweb-go/network"
+)
+
+// Config is used to configure the creation of the DNSProvider
+type Config struct {
+	URL                string
+	Username           string
+	Password           string
+	Zone               string
+	Timeout            time.Duration
+	PropagationTimeout time.Duration
+}
+
+// NewDefaultConfig returns a default configuration for the DNSProvider
+func NewDefaultConfig() *Config {
+	config := &Config{
+		URL:                env.GetOrDefaultString("LW_URL", ""),
+		Username:           env.GetOrDefaultString("LW_USERNAME", ""),
+		Password:           env.GetOrDefaultString("LW_PASSWORD", ""),
+		Zone:               env.GetOrDefaultString("LW_ZONE", ""),
```

Comment:
URL, Username, Password, zone musr be managed like other provider:

https://github.com/go-acme/lego/blob/e820634e0e43baff0ef61728cb8379f804913b72/providers/dns/cloudflare/cloudflare.go#L51-L63